Mathlib/Order/WellFoundedSet.lean

```diff
@@ -137,7 +138,9 @@ theorem acc_iff_wellFoundedOn {α} {r : α → α → Prop} {a : α} :
       WellFoundedOn { b | ReflTransGen r b a } r,
       WellFoundedOn { b | TransGen r b a } r] := by
   tfae_have 1 → 2 := by
-    refine fun h => ⟨fun b => InvImage.accessible _ ?_⟩
+    refine fun h => ⟨fun b => ?_⟩
+    unfold Subrel
```

Comment:
Doesn't this indicate that there's a missing lemma for subrel?

Reply:
I think it was just some odd behavior inferring arguments, I've managed to get rid of this `unfold`.